packages/fxa-auth-server/lib/sentry.js

```diff
@@ -0,0 +1,109 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const Hoek = require('@hapi/hoek');
+const Sentry = require('@sentry/node');
+
+const getVersion = require('./version').getVersion;
+
+// Matches uid, session, oauth and other common tokens which we would
+// prefer not to include in Sentry reports.
+const TOKENREGEX = /[a-fA-F0-9]{32}/gi;
```

Comment:
You could consider making this consume 32-or-more hex chars, so that longer tokens don't wind up looking like `[Filtered][Filtered]`.

Reply:
Sold! 😄